lib/linguist/languages.yml

```diff
@@ -5099,6 +5080,25 @@ Standard ML:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-ocaml
   language_id: 357
+Starlark:
+  type: programming
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#76d275"
+  extensions:
+  - ".bazel"
+  - ".bzl"
+  - ".starlark"
```

Comment:
Copybara uses *.sky as well. 

The Go Starlark interpreter examples uses *.star.

Reply:
I think `.sky` should be abandoned since Skylark is Starlark at this point. Gonna add `.star` since there are 20K hits and it might be a widespread extension in the future. Thanks!

Reply:
You'll need to remove `.starlark` because it falls short of our usage requirements (only [41 files](https://github.com/search?q=extension%3Astarlark+NOT+nothack&type=Code) at the time of writing). Otherwise, this looks file to me.